recipes/lpython/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "lpython" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lcompilers/lpython/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 690290b497315db3401ac626d0b5fdb8a2091fff538a57f664640fa16c2f1e73
+  patches:
+    - macos_clock.patch
+    - win_tests.patch
+    - msvc_static.patch
+    - version.patch
+
+build:
+  number: 0
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make   # [unix]
+    - xeus =2.4.1       # [build_platform != target_platform]
+    - llvmdev =15.0.7   # [build_platform != target_platform]
+    - xtl               # [build_platform != target_platform]
+    - nlohmann_json     # [build_platform != target_platform]
+    - cppzmq            # [build_platform != target_platform]
+    - zlib              # [build_platform != target_platform]
+    - bison =3.4        # [not win]
+    - m2-bison =3.0.4   # [win]
+    - xonsh =0.13.4     # [win]
+    - ninja             # [win]
+
+  host:
+    - re2c =2.2
+    - xeus =2.4.1
+    - llvmdev =11.1.0   # [build_platform == target_platform]
+    - llvmdev =15.0.7   # [build_platform != target_platform]
```

Comment:
We shouldn't do different versions depending on whether cross compiling or native. If you want to be explicit about `osx and arm64`, we should use that as the selector.

Reply:
I fixed this in 5f33d18e4517d827dd78908ae3bd4cfe2ced8299, since we don't need this in this PR. Once this is merged and I enable Apple arm64, there might be more bugs to fix, so I'll deal with it there.